projects/miner/codex_de839e3d/tools.py

```
@@ -0,0 +1,60 @@
+      }
+    }
+    """
+
+    r = []
+    try:
+        r = await request_codex({
+            "query": query,
+            "type": "getNetworks",
+            "variables": {},
+        })
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)